lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--pitch.tpl.php

```diff
@@ -27,14 +26,15 @@
   </header>
 
   <?php if (isset($campaign->secondary_call_to_action)): ?>
-    <div class="cta -persistent js-fixedsticky">
+    <div class="cta optimizely-hide-count -persistent js-fixedsticky">
```

Comment:
I think this should be re-ordered `cta -persistent optimizely-hide-count js-fixedsticky` so its clear that `-persistent` is modifying `cta`.